resources/views/layouts/navigation.blade.php

```diff
@@ -12,18 +12,38 @@
 
                 <!-- Navigation Links -->
                 <div class="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
+
                     <x-nav-link :href="route('dashboard')" :active="request()->routeIs('dashboard')">
                         {{ __('Dashboard') }}
                     </x-nav-link>
                     <x-nav-link :href="route('games')" :active="request()->routeIs('games')">
                         {{ __('Game List') }}
                     </x-nav-link>
+
+                    <div class="mb-2 mt-3 ml-3 xl:w-96">
+                        <div class="input-group relative flex flex-wrap items-stretch w-full mb-4">
+                            <input type="search" class="form-control relative flex-auto min-w-0 block w-full px-3 py-1.5 text-base font-normal text-black-700 bg-white bg-clip-padding border border-solid border-gray-500 rounded transition ease-in-out m-0 focus:text-black-700 focus:bg-white focus:border-blue-600 focus:outline-none" placeholder="Search" aria-label="Search" aria-describedby="button-addon2">
+
+                            <x-nav-link :href="route('results')" :active="request()->routeIs('search-results')" ><button class="btn inline-block px-6 py-4 bg-black-600 text-black font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-green-300 hover:shadow-lg focus:bg-blue-700  focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out flex items-center" type="button" id="button-addon2">
```

Comment:
The search button looks great! I think we need to put the search bar and button inside a form element and change the button to submit.